src/components/sideMenu/SideDrawerMenu.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react'
+import styled from 'styled-components'
+import SideMenuCloseControl from './SideMenuCloseControl'
+import { css } from 'styled-components'
+
+const SideDrawerMenu = ({open, onCloseMenu}) => (
+    <Wrapper open = {open}>
+        <SideMenuCloseControl onClick = {onCloseMenu}/>
```

Comment:
what happens onClick? closeMenu? or toggleMenu?
try to read the code like a conversation